Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -1354,6 +1354,34 @@ extension SourceKitServer {
     languageService.symbolInfo(request)
   }
 
+  func respondWithInterface(
+    _ req: Request<DefinitionRequest>,
+    moduleName: String,
+    symbol: String?,
+    languageService: ToolchainLanguageServer
+  ) {
+      var moduleName = moduleName
+      // Stdlib Swift modules are all in the "Swift" module, but their symbols return a module name `Swift.***`.
+      if moduleName.hasPrefix("Swift.") {
+        moduleName = "Swift"
+      }            
```

Comment:
Is there a better way to do this? Are there other modules that might be split up like this?

Reply:
To extend on @bnbarham’s comment, you can specify the group name by adding the following to the `editor.open.interface` request.

```
key.groupname: "String",
```

Reply:
I moved this code into the initialiser for OpenInterfaceRequest, made it slightly more generic and also set `key.groupname` when calling `editor.open.interface`. It appears to take an array of Strings so you can set ["Math", "Integers"] for sub-modules of sub-modules.